ai-ml/trainium-inferentia/variables.tf

```diff
@@ -37,3 +37,14 @@ variable "enable_amazon_prometheus" {
   type        = bool
   default     = true
 }
+
+variable "mpi_operator_version" {
+  description = "The version of the MPI Operator to install"
```

Comment:
Add type as string

Reply:
@vara-bonthu Thank you for the feedback. I added type as string in the variable